src/components/CourseChecklist/CourseChecklist.scss

```diff
@@ -0,0 +1,39 @@
+@import 'edx-bootstrap';
+
+//complete
+.checklist-item-complete {
+  box-shadow: -5px 0 0 0 $success;
+
+  &:hover {
+    box-shadow: -5px 0 0 0 theme-color("primary");
+  }
+}
+
+//incomplete
```

Comment:
Nit: for this to say something like "incomplete checklist style" or something along those lines, I read it as a comment to say that the code you wrote wasn't complete at first.